item-counter/test/App.test.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import { test, expect } from "@playwright/test";
+
+test.describe("item-counter", () => {
+	/**
+	 * Tracks the collaboration URL that is generated when navigating to the default URL.
+	 * Includes the generated container ID.
+	 */
+	let collaborationUrl: string | undefined;
+
+	test.beforeEach(async ({ page }) => {
+		await page.goto("http://localhost:8080", { waitUntil: "domcontentloaded" });
```

Comment:
This should work thanks to the config, right?
```suggestion
		await page.goto("/", { waitUntil: "domcontentloaded" });
```